src/containers/fluxHeatmap/FluxHeatmap.jsx

```diff
@@ -41,76 +40,73 @@ export default class FluxHeatmap extends React.Component<Props, State> {
 	state = {
 		container: {
 			height: 1,
-			width: 1
+			width: 1,
+			originalHeight: 1,
+			originalWidth: 1,
+			loaded: false
 		}
 	};
 
 	heatmap: Heatmap;
-	divElement: ?HTMLDivElement;
 	imgElement: ?HTMLImageElement;
 	setData: (ReadingModel[]) => void;
 	setConfig: ConfigObject => void;
 	setContainerState: () => void;
-	updateFromContainerState: () => void;
 
 	constructor(props: Props) {
```

Comment:
Constructor not needed with React 16 and ES6.
Redo the method declaration to match other containers with the lambda declaration `setData = () => {}`. You don't need to bind the `this` operator with lambda functions and it is recommended. You don't need to pass `props` to the `super()` constructor, if they are defined as the type of the `React.Component`.